src/routes/(admin)/tests/test-[type]/+page.svelte

```
@@ -2,44 +2,93 @@
-	let filteredTags: string[] = $state([]);
-	let filteredStates: string[] = $state([]);
+	import { DEFAULT_PAGE_SIZE } from '$lib/constants';
 
 	let {
 		data
 	}: {
 		data: {
 			test_taker_url: string;
 			is_template: boolean;
-			tests: any[];
+			tests: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)